using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md

```diff
@@ -1,56 +1,120 @@
 # Mobile Services API reference
 
-## Version of the Mobile Services extension
+## Process referrer intent
 
-The `extensionVersion()` API returns the version of the Mobile Services extension that is registered with the Mobile Core extension.
+You can use this API to process the referrer intent that was received from Android.
 
-To get the version of the Mobile Services extension, use the following code sample:
+{% hint style="warning" %}
+This API is **only** available in Android.
+{% endhint %}
 
-{% tabs %}
-{% tab title="Android" %}
-### Java
+### processReferrer
+
+**Syntax**
 
 ```java
-String mobileServicesExtensionVersion = MobileServices.extensionVersion();
+public static void processReferrer(final Context context, final Intent intent)
 ```
-{% endtab %}
 
-{% tab title="iOS — Obj-C" %}
-### Objective-C
+**Example**
 
-```objectivec
-NSString *mobileServicesExtensionVersion = [ACPMobileServices extensionVersion];
+**Java**
+
+```java
+public  void onReceive(Context context, Intent intent) {
+    MobileServices.processReferrer(context, intent);
+}
 ```
 
-### Swift
+## Process the Google Play Install Referrer
```

Comment:
Rename title to name of API `processGooglePlayInstallReferrerUrl` and move section above `processReferrer` for alphabetical listing of APIs.